src/containers/StudentDetailContainer/StudentDetailContainer.js

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Grid } from '@material-ui/core';
+import { getStudentById } from '../../store/actions/studentsAction';
+import StudentDetailItem from '../../components/StudentDetailItem/StudentDetailItem';
+
+const StudentDetailContainer = ({ id }) => {
+  const dispatch = useDispatch();
+  //const id = props.match.url.split('/')[3];
+  const studentById = useSelector((state) => state.students.student);
+
+  useEffect(() => {
+    dispatch(getStudentById(id));
+  }, [id]);
+
+  return (
+    <Grid container direction='column' justify='space-between'>
+      <StudentDetailItem label='Имя:' info={studentById.firstName} />
+      <StudentDetailItem label='Фамилия:' info={studentById.lastName} />
+      <StudentDetailItem label='Отчество:' info={studentById.middleName} />
+      <StudentDetailItem label='Класс:' info={studentById.grade} />
+      <StudentDetailItem label='Язык:' info={studentById.language} />
+      <StudentDetailItem label='Школа:' info={studentById.school} />
+      <StudentDetailItem label='Контакты родителя(ей):' info={studentById.parentsContacts} />
+      <StudentDetailItem label='Поток:' info={studentById.stream} />
```

Comment:
Изменить Поток на Направление, stream на streamId
```suggestion
      <StudentDetailItem label='Направление:' info={studentById.streamId} />
```

Reply:
@xx3u  Изменен